bob.js

```diff
@@ -1064,13 +1064,13 @@ recordResults = () => {
                 }
             } else if (p1Place < p2Place) {
                 // Ahead of opponent, count as win
-                actualScore += 1;
+                actualScore++;
             } else {
                 // Loss gives 0 points
             }
         });
 
-        newElos.set(id1, playerStats.get(id1).elo + 32 * (actualScore - expectedScore));
```

Comment:
Keep as is for readability. Also makes potential tuning in the future easier since we're not restricted to powers of 2.